### PR TITLE
Word2007 Writer : Fixed first footnote appearing as separator

### PR DESCRIPTION
**Pull request by jacksleight**
### Description

Fixes https://github.com/PHPOffice/PHPWord/issues/2634

This issue is caused by conflicting `w:id`s in the footnote data:

```xml
<w:footnote w:id="0" w:type="continuationSeparator">
  <w:p>
    <w:r>
      <w:continuationSeparator/>
    </w:r>
  </w:p>
</w:footnote>
<w:footnote w:id="0">
  ...
</w:footnote>
```

The continuation separator and the first footnote have the same `w:id`.

I tested how Word saves these and the footnotes start with a `w:id` of 1 not 0, so I just added that offset. I don't know if that's necessarily the best way to fix it (this is my first PHPWord PR), can make any changes if needed.

I also added new assertions to the footnotes test, to check that a footnote with `w:id` 0 does not exist and 1 does, but again, not sure if this is the best way to do that check.

### Checklist:

- [x] My CI is :green_circle:
- [x] I have covered by unit tests my new code (check build/coverage for coverage report)
- [ ] I have updated the [documentation](https://github.com/PHPOffice/PHPWord/tree/master/docs) to describe the changes
- [x] I have updated the [changelog](https://github.com/PHPOffice/PHPWord/blob/master/docs/changes/2.x/2.0.0.md)

### CODE REVIEW

File: tests/PhpWordTests/Writer/Word2007/Part/FootnotesTest.php [hunk 1]
```diff
@@ -49,5 +49,8 @@ public function testWriteFootnotes(): void
 
         self::assertTrue($doc->elementExists('/w:document/w:body/w:p/w:r/w:footnoteReference'));
         self::assertTrue($doc->elementExists('/w:document/w:body/w:p/w:r/w:endnoteReference'));
+
+        self::assertFalse($doc->elementExists('/w:document/w:body/w:p/w:r/w:footnoteReference[@w:id="0"]'));
+        self::assertTrue($doc->elementExists('/w:document/w:body/w:p/w:r/w:footnoteReference[@w:id="1"]'));
```
Comment (Progi1984) [inline tests/PhpWordTests/Writer/Word2007/Part/FootnotesTest.php:54]:
```suggestion
        self::assertFalse($doc->elementExists('/w:document/w:body/w:p/w:r/w:footnoteReference[@w:id="0"]'));
        self::assertTrue($doc->elementExists('/w:document/w:body/w:p/w:r/w:footnoteReference[@w:id="1"]'));

        self::assertFalse($doc->elementExists('/w:document/w:body/w:p/w:r/w:endnoteReference[@w:id="0"]'));
        self::assertTrue($doc->elementExists('/w:document/w:body/w:p/w:r/w:endnoteReference[@w:id="1"]'));
```

**Comment by Progi1984** [comment]
@jacksleight Hi, thank you for your contribution.

Could you update the [2.0.0.md changelog](https://github.com/PHPOffice/PHPWord/blob/master/docs/changes/2.x/2.0.0.md), please ?

**Comment by jacksleight** [comment]
@Progi1984 Ah yeah, that's done.

File: docs/changes/2.x/2.0.0.md [hunk 2]
```diff
@@ -25,6 +25,7 @@
 - Documentation : Updated Comment element by [@laminga](https://github.com/laminga) in [#2650](https://github.com/PHPOffice/PHPWord/pull/2650)
 - HTML Reader : Read width & height attributes in points fixing [#2589](https://github.com/PHPOffice/PHPWord/issues/2589) by [@Progi1984](https://github.com/Progi1984) in [#2654](https://github.com/PHPOffice/PHPWord/pull/2654)
 - Template Processor : Fixed bad naming of variables fixing [#2586](https://github.com/PHPOffice/PHPWord/issues/2586) by [@Progi1984](https://github.com/Progi1984) in [#2655](https://github.com/PHPOffice/PHPWord/pull/2655)
+- MsDoc Writer : Fix first footnote appearing as separator [#2634](https://github.com/PHPOffice/PHPWord/issues/2634) by [@jacksleight](https://github.com/jacksleight) in [#2635](https://github.com/PHPOffice/PHPWord/pull/2635)
```
Comment (Progi1984) [inline docs/changes/2.x/2.0.0.md:28]:
```suggestion
- Word2007 Writer : Fix first footnote appearing as separator [#2634](https://github.com/PHPOffice/PHPWord/issues/2634) by [@jacksleight](https://github.com/jacksleight) in [#2635](https://github.com/PHPOffice/PHPWord/pull/2635)
```

**Review by Progi1984 — COMMENTED** [review]
A last comment

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thank you @jacksleight for your contribution

**Pull request merged**